python/private/python.bzl

```diff
@@ -357,6 +357,13 @@ def _process_single_version_overrides(*, tag, _fail = fail, default):
             for platform in sha256
         }
 
+    if tag.flag_values:
+        # Normalize flag keys to strings
+        flag_values = {str(k): v for k, v in tag.flag_values.items()}
+        override["flag_values"] = flag_values
+    if tag.suffix:
+        override["suffix"] = tag.suffix
```

Comment:
The `tag.suffix` should be used to construct `toolchain_info.name` so that a `3.13.0` version with and without free threading can coexist together.

I am under an assumption that these will be two separate `tar.gz` files that get downloaded - one with and one without free threading support, am I right?